src/renderers/webgl/WebGLRenderStates.js

```diff
@@ -54,26 +61,26 @@ function WebGLRenderStates() {
 
 	let renderStates = new WeakMap();
 
-	function get( scene, camera ) {
+	function get( scene, renderCallNesting ) {
```

Comment:
`renderCallNesting` sounds not descriptive enough. In some sense, it represents the depth or level of nested render calls. Maybe just `level`?

Reply:
How about `renderCallDepth` or `renderCallLevel`? `level` by itself doesn't mean much.

Reply:
Then I vote for `renderCallDepth`.